Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -360,21 +370,27 @@ public class HTTPServer: Server {
 
      ### Usage Example: ###
      ````swift
-     let server = HTTPServer.listen(on: 8080, delegate: self)
+     let server = HTTPServer.listen(on: 8080, node: "localhost", delegate: self)
      ````
 
      - Parameter on: Port number for accepting new connections.
+     - Parameter node: has the same meaning as in `getaddrinfo()`
      - Parameter delegate: The delegate handler for HTTP connections.
 
      - Returns: A new instance of a `HTTPServer`.
     */
-    public static func listen(on port: Int, delegate: ServerDelegate?) throws -> ServerType {
+    public static func listen(on port: Int, node: String?, delegate: ServerDelegate?) throws -> ServerType {
```

Comment:
Can we have the `node`/`address` parameter have a default value of `nil`? To make sure this is not a breaking change.